exercise/static/exercise/results_staff.js

```diff
@@ -0,0 +1,840 @@
+let currentScript = (document.currentScript ?
+    $(document.currentScript) :
+    $('script').last()); // Ugly solution for IE11
+
+let allExercisesUrl = currentScript.attr("allExercisesUrl");
+let exercisesUrl = currentScript.attr("exercisesUrl");
+let studentsUrl = currentScript.attr("studentsUrl");
+let pointsUrl = currentScript.attr("pointsUrl");
+let usersUrl = currentScript.attr("usersUrl");
+let usertagsUrl = currentScript.attr("usertagsUrl");
+let taggingsUrl = currentScript.attr("taggingsUrl");
+let externalLabel = currentScript.attr("externalLabel");
+let internalLabel = currentScript.attr("internalLabel");
+
+let _exerciseSelection;
+let _allExercises = [];
+let _exercises;
+let _students;
+let _users = {};
+let _usertags;
+let _taggings;
+let _points = {};
+
+let tableExportVar;
+// TableExport plugin
+// https://tableexport.v5.travismclarke.com/#tableexport
+
+$(document).on("aplus:translation-ready", function() {
+    TableExport.prototype.formatConfig.xlsx.buttonContent = _("Export to xlsx (Excel)");
+    TableExport.prototype.formatConfig.csv.buttonContent = _("Export to csv (LibreOffice)");
+    TableExport.prototype.formatConfig.txt.buttonContent = _("Export to txt");
+    tableExportVar = $("#table-points").tableExport({
+        "position": "top",
+        "bootstrap": "true",
+        "filename": _("student_results"),
+    });
+
+// Move the default TableExport download buttons inside a single dropdown menu
+    $("caption.tableexport-caption").children("button").each(function() {
+        $("#export-button-menu").append($(this).detach());
+    });
+});
+
+// Make the table headings and student IDs stick when scrolling table
+$('#table-points-div').scroll(function(ev) {
+    $('thead#table-heading th').css('transform', 'translateY(' + this.scrollTop + 'px)');
+    $('tbody td.stick-on-scroll').css('transform', 'translateX(' + this.scrollLeft + 'px)');
+});
+
+
+// Booleans to allow teacher to choose with checkboxes what data indicators they want to see
+let _totalSubmTrue = false;
+let _avgSubmTrue = false;
+let _maxSubmTrue = false;
+let _totalStuSubmTrue = false;
+let _totalStuMaxTrue = false;
+let _avgPTrue = false;
+let _maxPTrue = false;
+
+$("#total-subm-checkbox").change(function() {
+    _totalSubmTrue = this.checked ? true : false;
```

Comment:
`_totalSubmTrue = this.checked ? true : false;`
Is this any different from simply assigning like this `_totalSubmTrue = this.checked;`

Reply:
It is same. Currently using `_totalSubmTrue = this.checked;`